specs/app.spec.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs'),
```

Comment:
We should add tests for this file for append and overwrite, and also using append and overwrite with the files option. The test files should be well named and thought through. For example, for the append tests you could have 2 files; `specs/data/transactions/may/transactions.csv` and `specs/data/transactions/june/transactions.csv`. You could first insert the file in the may folder and then the file in the june folder. This way they have the same file name but different contents.